spaceship/lib/spaceship/two_step_client.rb

```diff
@@ -35,7 +35,7 @@ def handle_two_step(response)
         result = choose(*available)
         device_id = result.match(/.*\t.*\t\((.*)\)/)[1]
         select_device(r, device_id)
-      elsif r.body.kind_of?(Hash) && r.body["phoneNumberVerification"].kind_of?(Hash)
+      elsif r.body.kind_of?(Hash) && r.body["trustedPhoneNumbers"].kind_of?(Array) && r.body["trustedPhoneNumbers"].first.kind_of?(Hash)
```

Comment:
What is the importance of checking `"trustedPhoneNumbers"` here? It doesn't appear that we ever use the value, or the hash it contains.

Also, should we be checking the existence/type of `response.body["securityCode"]`, which is now used on line 51 below?